### ISSUE TITLE
How can split ImageNet dataset into train and validation set ?

### ISSUE THREAD

**Issue by ma-kjh** [locuslab/FLYP#22]
Thank you for your great work.

I have a question for splitting train and validation dataset in ImageNet-1k dataset.

In the paper, you mentioned 80:20 train:val, but the code (ImageNet) was not splitted.

How can I reproduce FLYP imageNet 82.6% accuracy (OpenAI CLIP ViT-B/16) ?

![Screenshot 2024-03-11 at 16 32 53](https://github.com/locuslab/FLYP/assets/132344612/7f629884-4873-426e-a33b-d3654ce43bde)

**Comment by SachinG007** [comment]
Hi @ma-kjh ,
For ImageNet, we indeed ended up training using the whole data and reporting the best test accuracy, which always is the last checkpoint. For imagenet, we never observed an overfitting phenomenon where the test accuracy was higher at some intermediate stage. 

To reproduce the numbers, you can use the scripts in the README
